tests/resources/VCH-Util.robot

```diff
@@ -201,6 +201,7 @@ Run VIC Machine Command
 Run Secret VIC Machine Delete Command
     [Tags]  secret
     [Arguments]  ${vch-name}
+    Run  govc vm.unregister ${vch-name}
```

Comment:
Why do you need to unregister the VCH here?

Reply:
was trying to fix another issue but mhagen informed me there's more to it (I misunderstood him in the first place, in thinking this line was all that was needed) so I just removed this